media/js/privacy/privacy-protocol.js

```diff
@@ -19,3 +19,16 @@
         buttons[i].setAttribute('data-close', closeText);
     }
 })();
+
+if (window.location.href.includes('cookies')) {
```

Comment:
Suggest using `window.location.hash` here to be a little more explicit:
```suggestion
if (window.location.hash === '#cookies') {
```

Reply:
Good suggestion, thank you!